### ISSUE TITLE
Incorrect URL in the example packer template

### ISSUE THREAD

**Issue by blaisep** [hashicorp/terraform-google-consul#8]
The URL in the sample packer template at:
https://github.com/hashicorp/terraform-google-consul/tree/master/examples/consul-image#quick-start is:
https://github.com/gruntwork-io/consul-gcp-module.git
which returns 404.
I believe the correct URL should be:
https://github.com/hashicorp/terraform-google-consul.git

**Comment by blaisep** [comment]
I updated the `provisioners` block in https://github.com/hashicorp/terraform-google-consul/tree/v0.0.1/examples/consul-image/README.md
to:
```
  "provisioners": [{
    "type": "shell",
    "inline": [
        "git clone --branch v0.0.1 https://github.com/hashicorp/terraform-google-consul.git",
        "/tmp/consul-gcp-module/modules/install-consul/install-consul --version {{user `consul_version`}}",
        "/tmp/consul-gcp-module/modules/install-dnsmasq/install-dnsmasq"
    ]
```
and it worked for me.

**Comment by blaisep** [comment]
I can commit the corrected file with a PR, if that makes things easier.

**Comment by josh-padnick** [comment]
@blaisep You're correct that this is a typo. A PR would be a very welcome fix!

By the way, the long-term plan here is to add automated tests to this repo to guarantee that there are no typo error like that preventing things from working.

**Comment by blaisep** [comment]
Hi @josh-padnick thanks for the reality check, I will clean up the README example. Also, there is a reference to the none-existent `Releases` page, and I wonder if it is better (for now) to instruct the reader to use the tags in the repo, eg:
_To install Consul, use `git clone --branch` to clone this repository at a specific tag, then run the `install-consul` script_ :
```
git clone --branch <MODULE_VERSION>
https://github.com/hashicorp/terraform-google-consul.git
terraform-google-consul/modules/install-consul/install-consul --version 1.0.6
```

**Comment by josh-padnick** [comment]
Fixed in #9
